api/common/v1beta2/common_types.go

```diff
@@ -15,6 +17,8 @@ type KubernetesConfig struct {
 	ImagePullSecrets                     *[]corev1.LocalObjectReference                          `json:"imagePullSecrets,omitempty"`
 	UpdateStrategy                       appsv1.StatefulSetUpdateStrategy                        `json:"updateStrategy,omitempty"`
 	PersistentVolumeClaimRetentionPolicy *appsv1.StatefulSetPersistentVolumeClaimRetentionPolicy `json:"persistentVolumeClaimRetentionPolicy,omitempty"`
+	AdditionalVolumes                    []corev1.Volume                                         `json:"additionalVolumes,omitempty"`
```

Comment:
remove AdditionalVolumes/AdditionalVolumeMounts and related codes. or raise another pr to implement ACLConfig.PersistentVolumeClaim :)